modules/version-update/pages/update-with-migration-tool.adoc

```diff
@@ -270,7 +270,7 @@ For example, db.url=jdbc:mysql://localhost:3306/bonita-update?allowMultiQueries=
 [WARNING]
 ====
 
-*IMPORTANT:* Do *not* xref:ROOT:pause-and-resume-bpm-services.adoc[pause the BPM services] before you stop the application server, unless your Bonita source version is higher than 2021.1, otherwise it will cause problems.
+*IMPORTANT:* Do *not pause the BPM services* (via maintenance mode) before you stop the application server, unless your Bonita source version is higher than 2021.1, otherwise it will cause problems.
```

Comment:
```suggestion
*IMPORTANT:* Do *not pause the BPM services* (or enable the maintenance mode) before you stop the application server, unless your Bonita source version is higher than 2021.1, otherwise it will cause problems.
```